docs/docs/how-tos/nebari-aws.md

```diff
@@ -94,23 +94,23 @@ installed in your environment, (iii) opted for **AWS** as your cloud provider, a
 installation, please visit the "Getting Started" section of our [troubleshooting page](/troubleshooting) for further guidance.
 :::
 
-## Nebari Initialize
+## Initializing Nebari
 
 Great, you’ve gone through the [Nebari Installation](/getting-started/installing-nebari.md) and [authentication setup](#authentication) steps, and have ensured that all the necessary
-environment variables have been properly set. It is time to initialize and deploy Nebari!
+environment variables have been properly set.
+
+In this step, you'll run `qhub init` to create the `qhub-config.yaml`.
 
 1. In your terminal, start by creating a new project folder. For this demonstration, we will name the new folder `nebari-aws`:
 
    ```bash
    mkdir nebari-aws && cd nebari-aws
    ```
 
-2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
+2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**. The `projectname` will be the name of the folder/repo that will manage this Nebari deployment (it will be created). The `domain` will the domain endpoint for your Nebari instance. The `auth-provider` sets your authentication provider that you plan to use inside of Keycloak, options are Github, Auth0, and password. For this example, we'll run with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**. These can be updated later by directly modifying the `qhub-config.yaml`.
 
 ```bash
-qhub init aws --project projectname \
-	  --domain domain \
-	  --auth-provider password
+qhub init aws --project projectname --domain domain --auth-provider password
```

Comment:
Using `\` and multiline makes it easier to read the command in smaller screens so I'd prefer leaving this as it was before

Reply:
for people actually following along with this as a guide, they will copy and paste the command. Its very frustrating for this to be multi-line in that case. ( I was frustrated by it when I walked through the guide and Eskild mentioned he found it annoying too)

Reply:
the `\` makes it easier to read multiline on smaller screens but harder to read on larger screens. So we are targeting the minority. 

This is what it looks like on my laptop 
<img width="1663" alt="image" src="https://user-images.githubusercontent.com/15659110/192541555-7135cfa1-fc19-40ca-9f95-3a341ff721bc.png">

Reply:
All that said, I defer to your final decision.

Reply:
> the \ makes it easier to read multiline on smaller screens but harder to read on larger screens. So we are targeting the minority.

Disagree on targeting the minority, smaller screens also count as wide screens being split in thirds/quarters or long format screens. 
Others need to use zoom for low vision concerns which also lead to inaccessible long strings of text, not to mention those who navigate sites with a keyboard only.


Re accessibility - the natural text in (the body of) websites usually has automatic reflow, this is not directly applied to `<pre> <code>`  elements, so splitting the long string here helps as a workaround for the lack of automatic reflow.

I do not think it makes it harder to read on larger screens - it is just a different display format to what some might be used to and many other documentation sites do use this split syntax for long commands, often with many flags. So I prefer keeping this.

Reply:
Also about this in particular

> So we are targeting the minority.
I have very strong feelings about this - it is usually minorities that get underserved / not thought of. So I prefer thinking in terms of the "the-curb-cut-effect" more often than not

Reply:
I apologize for using a trigger word. I didn't mean it in that sense. Backing away slowly now....